buildenv/docker/mkdocker.sh

```diff
@@ -671,7 +674,7 @@ print_dockerfile() {
 if [ $cuda != no ] ; then
   install_cuda
 fi
-  install_freemarker
+  #install_freemarker
```

Comment:
I'd prefer to see this controlled by an option, say, `--freemarker`.

Reply:
Added to this latest commit.